src/app/components/register/register.component.html

```diff
@@ -0,0 +1,144 @@
+<div class="registerPage">
+  <div class="form">
+    <form (ngSubmit)="register()" [formGroup]="registerForm">
+      <p-panel class="content" header="register">
+        <div class="form-group">
+          <div class="p-field">
+            <label for="email">Email*</label>
+            <input formControlName="email" pInputText/>
+            <p-message *ngIf="registerForm.controls['email'].invalid
+                        &&registerForm.controls['email'].dirty" severity="error" text="Email is required">
+            </p-message>
```

Comment:
This can be done in the component.ts
You can call form.valid and then mark for as dirty with form.markAsDirty

Reply:
Will be refactored later